granite.gemspec

```diff
@@ -13,12 +13,12 @@ Gem::Specification.new do |s|
   s.files       = `git ls-files`.split("\n").grep(/\A(app|lib|config|LICENSE)/)
   s.license     = 'MIT'
 
-  s.add_runtime_dependency 'actionpack', '>= 5.1', '< 6.1'
+  s.add_runtime_dependency 'actionpack', '>= 5.1', '< 7'
```

Comment:
Looks bad without check for rails+active_data versions :/
Right now granite+rails-6.1 would just fail in unexpected places without knowledge that active_data from some-non-upstream git repo should be used.